src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -2383,53 +2384,48 @@ public void run() {
   public final static int pearlId = 368;
   public final static MinecraftKey pearlKey = new MinecraftKey("ender_pearl");
 
-  @SuppressWarnings({ "rawtypes", "unchecked" })
+  @SuppressWarnings("unchecked")
   @BahHumbug(opt="ender_pearl_gravity", type=OptType.Double, def="0.060000")
-  private void hookEnderPearls() {
+  public void hookEnderPearls() {
```

Comment:
Honestly, just leave the version at 1.10.0 -- no compiles have occurred at that version level, no need to incrtement it.

Then you don't need to change any others you've already done.